src/server.jsx

```diff
@@ -70,7 +70,7 @@ export default locals => {
           <meta name="apple-mobile-web-app-capable" content="yes" />
           <meta name="apple-mobile-web-app-status-bar-style" content="black" />
           <meta name="apple-mobile-web-app-title" content="webpack" />
-          <link rel="apple-touch-icon-precomposed" href="/icon_180x180.png" />
```

Comment:
Updated according to https://web.dev/apple-touch-icon/

![image](https://user-images.githubusercontent.com/1091472/94269926-ecf3a800-ff71-11ea-9a0a-659eabf2ce0f.png)